src/CTA.WebForms2Blazor/ProjectManagement/ProjectBuilder.cs

```diff
@@ -33,21 +41,28 @@ public void WriteFileInformationToProject(FileInformation newDocument)
 
         public void WriteFileBytesToProject(string relativePath, byte[] fileContent)
         {
-            CreateRelativeDirectoryIfNotExists(Path.GetDirectoryName(relativePath));
-            var fullPath = Path.Combine(_outputProjectPath, relativePath);
-
-            if (File.Exists(fullPath))
+            try
             {
-                LogHelper.LogInformation(string.Format(FileAlreadyExistsLogTemplate, GetType().Name, fullPath));
-                // TODO: Maybe copy and store the old version of this file somehow?
-            }
+                CreateRelativeDirectoryIfNotExists(Path.GetDirectoryName(relativePath));
+                var fullPath = Path.Combine(_outputProjectPath, relativePath);
+
+                if (File.Exists(fullPath))
+                {
+                    LogHelper.LogInformation(string.Format(FileAlreadyExistsLogTemplate, GetType().Name, fullPath));
+                    // TODO: Maybe copy and store the old version of this file somehow?
+                }
+
+                using (FileStream stream = File.Create(fullPath))
+                {
+                    stream.Write(fileContent, 0, fileContent.Length);
+                }
 
-            using (FileStream stream = File.Create(fullPath))
+                LogHelper.LogInformation(string.Format(FileInfoWrittenLogTemplate, GetType().Name, fullPath));
+            }
+            catch (Exception e)
             {
-                stream.Write(fileContent, 0, fileContent.Length);
+                LogHelper.LogError(e, "Could not write file bytes to project.");
```

Comment:
May want to include the file location we attempted to write to here

Reply:
The path might be recorded by the exception itself, but will add here explicitly just in case